app/src/main/java/de/tum/in/tumcampusapp/activities/CalendarActivity.java

```diff
@@ -416,9 +419,11 @@ public String interpretTime(int hour) {
 
     @Override
     public void onEventClick(WeekViewEvent weekViewEvent, RectF rectF) {
-        IntegratedCalendarEvent event = (IntegratedCalendarEvent) weekViewEvent;
-        Intent i = new Intent(this, RoomFinderDetailsActivity.class);
-        i.putExtra(RoomFinderDetailsActivity.EXTRA_LOCATION, event.getLocation());
-        this.startActivity(i);
+        CalendarDetailsFragment detailsFragment = new CalendarDetailsFragment();
+        Bundle bundle = new Bundle();
+        CalendarItem item = calendarManager.getCalendarItemByStartAndEndTime(weekViewEvent.getStartTime(), weekViewEvent.getEndTime());
+        bundle.putString(CALENDAR_ID_PARAM, item.getNr());
+        detailsFragment.setArguments(bundle);
+        detailsFragment.show(getSupportFragmentManager(), "CalendarDialog");
```

Comment:
That hardcoded string seems fishy. Do we need that tag, or can't we just call `show` with a null tag?